processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/parallelAlgorithm2.asm (1 of 2)
processing file asmFiles/parallelAlgorithm.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
--- memsim.hex	2016-04-19 21:05:46.085621208 -0400
+++ memcpu.hex	2016-04-18 23:35:29.159916050 -0400
@@ -1,23 +1,24 @@
-:0400000034080400BC
-:040001003C09DEAD2B
-:040002003529BEEFEF
-:04000300AD09000043
-:040004003408040CAC
-:040005003C09CAB137
-:040006003529FEEDAD
-:04000700AD0900003F
-:04000800FFFFFFFFF8
-:040080003408040438
-:040081003C0989AB02
-:040082003529CDEF60
-:04008300AD090000C3
-:040084003408040830
-:040085003C0901230E
-:04008600352945676C
-:04008700AD090000BF
-:04008800FFFFFFFF78
-:04010000DEADBEEFC3
-:0401010089ABCDEF0A
-:040102000123456729
-:04010300CAB1FEED92
+:040000003408040CB0
+:040001008D09000065
+:0400020001204C008D
+:04000300340C0408AD
+:04000400340A0400B6
+:040005008D4B00001F
+:040006001160FFFE88
+:040007008D880000E0
+:040008000109402585
+:04000900AD880000BE
+:04000A00FFFFFFFFF6
+:040080003408041428
+:040081008D090000E5
+:04008200340A040830
+:04008300AD49000083
+:040084003409040037
+:04008500340A000138
+:04008600AD2A00009F
+:04008700FFFFFFFF79
+:0401000000000001FA
+:04010200DEADBEEFC1
+:040103000000DEAD6D
+:040105000000BEEF49
 :00000001FF
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/parallelAlgorithm.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/parallelAlgorithm.asm (1 of 1)
